src/enum.cr

```diff
@@ -474,4 +474,18 @@ struct Enum
       {% end %}
     {% end %}
   end
+
+  def self.all_flags : self
+    {% if @type.annotation(Flags) %}
+      {% value = 0 %}
+      {% for member in @type.constants %}
+        {% if member.stringify != "All" %}
+          {% value |= @type.constant(member) %}
+        {% end %}
+      {% end %}
+      new({{ value }})
+    {% else %}
+      {% raise "#{@type}.all_flags is only available on Flags Enum" %}
+    {% end %}
+  end
```

Comment:
Hm what's the use case for this `all_flags`?

Reply:
It's a replacement for the implicit `All` member.